src/main/java/br/com/sbk/sbking/core/boardrules/bridgeopenings/DealerHasOneNoTrumpOpeningBoardRule.java

```diff
@@ -0,0 +1,25 @@
+package br.com.sbk.sbking.core.boardrules.bridgeopenings;
+
+import br.com.sbk.sbking.core.Board;
+import br.com.sbk.sbking.core.BoardRule;
+import br.com.sbk.sbking.core.Hand;
+import br.com.sbk.sbking.core.HandEvaluations;
+
+public class DealerHasOneNoTrumpOpeningBoardRule implements BoardRule {
+
+  @Override
+  public boolean isValid(Board board) {
+    Hand dealerHand = board.getHandOf(board.getDealer());
+    HandEvaluations handEvaluations = dealerHand.getHandEvaluations();
+    return hasCorrectHCPRange(handEvaluations) && hasCorrectDistribution(handEvaluations);
+  }
+
+  private boolean hasCorrectHCPRange(HandEvaluations handEvaluations) {
+    return handEvaluations.getHCP() >= 15 && handEvaluations.getHCP() <= 17;
+  }
+
+  private boolean hasCorrectDistribution(HandEvaluations handEvaluations) {
+    return handEvaluations.isBalanced() && !handEvaluations.hasFiveCardMajorSuit();
```

Comment:
`hasFiveCardMajorSuit` means has five or more cards in major suit? .

Reply:
Yes. Renamed it to improve legibility.